.github/workflows/codeql.yml

```diff
@@ -0,0 +1,94 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
```

Comment:
Do you think adding workflow_dispatch here could be useful? I usually like it when I can manually trigger an action, not sure if that's necessary in its case though.

Reply:
Generally, I agree, but in this case I don't think we will ever do it. We have never needed it in SAP Cloud SDK. Also, same argument as above, I would like to touch as little as possible, because this file was generated.

Reply:
Yeah, I agree.